R/xp_utils.R

```diff
@@ -83,10 +83,9 @@ xp_call_name <- function(expr, depth = 1L, condition = NULL) {
   )
   is_valid_expr <- is_node(expr) || is_nodeset(expr)
   if (!is_valid_expr) {
-    stop(
-      "Expected an xml_nodeset or an xml_node, instead got an object of class(es): ",
-      toString(class(expr)),
-      call. = FALSE
+    cli_abort(
+      "{.arg expr} must be an {.cls xml_nodeset} or an {.cls xml_node}, \\
```

Comment:
ah this is a weird feature of the parser I'd rather not use TBH, I've not seen it documented (e.g. it's not in `?Quotes`) and don't see it used very often.

Reply:
I am not sure what "this" refers to here. The use of `\\`?